lib/rx-http.js

```diff
@@ -2,8 +2,9 @@ const httpRx = {};
 const request = require('request');
 const Observable = require('rxjs').Observable;
 const HTTP_STATUS_CODES = require('./http-status-codes');
+const HTTP_METHODS = require('./http-methods');
 
-const isCorrectHttpMethod = method => /get|head|patch|post|put|delete/i.test(method);
+const isCorrectHttpMethod = method => RegExp(HTTP_METHODS.join('|'), 'i').test(method);
```

Comment:
I propose to move `isCorrectHttpMethod` to another file and cover with unit tests (6 positive and 1 negative).